src/client/table_client.rs

```diff
@@ -541,34 +548,57 @@ impl ObTableClientInner {
         Ok(())
     }
 
+    /*
+     * get_partition_leaders will return gt part ids
+     */
```

Comment:
Maybe we should follow the style of the rust doc comments.

Reply:
Fixed